FreeTubeApp/FreeTube
# Make video info section more concise

# Make video info section more concise




## Pull Request Type


## Related issue
closes #4336 

## Description
- Puts title on its own row
- Puts video info section on its own row
- Puts subscribe button & video buttons on their own row
- Fixes bug of `videoOptions` buttons overflowing the video info section that occurred on some viewport widths

## Video 

https://github.com/FreeTubeApp/FreeTube/assets/84899178/18ce83a6-c7e9-4001-9fe5-d6e6ea4a3711



## Testing 
- Ensure that appearance is great at all screen sizes
- Ensure that the video options buttons no longer extend past the container when resizing viewport

## Desktop

- **OS:** OpenSUSE Tumbleweed
- **OS Version:** 2023xxxx
- **FreeTube version:** 0.19.1

## Additional context
